docs/introduction/foundations/components/integrations/connected-account.mdx

```diff
@@ -99,11 +98,13 @@ else:
 print("The account status is: ",connection_request)
 ```
 ```javascript javascript
-// Collect the subdomain from your user via your platform's frontend
-const params = { subdomain: "***zendesk_sub_domain***" };
+const entity = await client.getEntity({ id: "default" });
```

Comment:
Feels like this is wrong, "github" doesn't ask for these parameters - maybe use the same example as python (i.e "zendesk")